metadata.json

```diff
@@ -14,5 +14,8 @@
     {
       "operatingsystem": "Ubuntu"
     }
+  ],
```

Comment:
This is unnecessary, right?

Reply:
The dependencies key is indeed required.. see https://puppet.com/docs/puppet/7.1/modules_metadata.html

Reply:
I would say so, otherwise I wouldn't put it there.

Reply:
Also, please note this is 3 years old merge request, which I am not able to support anymore as I don't remember this kind of details.